src/Ocelot.Provider.Polly/PollyPoliciesDelegatingHandler.cs

```diff
@@ -1,10 +1,13 @@
+using System.Diagnostics;
+
```

Comment:
Please, don't shuffle usings!
Do you use Visual Studio or other IDE? Which one?

Reply:
This is due to our corporate coding rules, and therefore to my VS22 setting (system first, then microsoft, then the rest).

Reply:
same here! 😹